datadog_checks_dev/datadog_checks/dev/tooling/commands/manifest.py

```diff
@@ -67,21 +60,27 @@ def manifest():
 def verify(fix, include_extras):
     """Validate all `manifest.json` files."""
     all_guids = {}
-    failed = 0
+    task_failed = False
```

Comment:
The linked comment used as motivation was correct for that pr but does not apply to this command. Here we use the numeric value to keep track of fixes